src/test/scala/org/broadinstitute/dsde/workbench/sam/api/StandardSamUserDirectivesSpec.scala

```diff
@@ -253,6 +253,11 @@ class StandardSamUserDirectivesSpec extends AnyFlatSpec with PropertyBasedTestin
     }
   }
 
+  it should "not give an invited user a new user id" in {
+    val standardSamUserDirectives = directives()
+//    standardSamUserDirectives.buildSamUser(OIDCHeaders())
+  }
+
```

Comment:
We should flesh out this test or remove it.  If we remove it, we can undo the refactor of moving `buildSamUser` too.